### ISSUE TITLE
macos server

### ISSUE THREAD

**Issue by daniellaera** [b4rtaz/voice-assistant#1]
is would be nice to have a macos server for testing

**Comment by isidorn** [comment]
Agree. Having this cross platform would be great. We also have bunch of linux vscode users that could use this.
